src/opensignals/__main__.py

```diff
@@ -24,8 +24,7 @@
 from docopt import docopt
 
 from opensignals import __version__
-from opensignals.data import yahoo
-from opensignals.features import RSI
+from opensignals.data.yahoo import Yahoo
```

Comment:
I prefer importing only modules and packages, not individual classes. That makes it more obvious where something is coming from in the code. Also compare google*s python style guide with examples and why they prefer organizing imports that way: https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Thanks for the link, I didn't knew the Google style guide. 

I personally find it better to import only the things you need. Especially in dynamic languages which have limited support for dead code detection and refactoring I thinks it's important to see why and which part of a library you're using a specific part in your code.

In this case I don't think it is that important, I'm happy to conform to whatever style is desired.